src/main/java/org/traccar/protocol/WialonProtocolDecoder.java

```diff
@@ -156,11 +154,54 @@ private Position decodePosition(Channel channel, SocketAddress remoteAddress, St
                     }
                 }
             }
+
+            // interpret accuracy
+            if (position.hasAttribute("accuracy")) {
+                position.setAccuracy(position.getDouble("accuracy"));
+                position.removeAttribute("accuracy");
+            }
+
+            // interpret LBS
+            // this dance is necessary to account for the fact that the LBS data can be in multiple formats
+            Network network = new Network();
+            if (position.hasAttribute("mcc")) {
+                interpretLBSAndRemoveAttribute(position, network, "");
+            }
+            if (position.hasAttribute("mcc1")) {
+                for (int i = 1; i <= 9; i++) {
+                    interpretLBSAndRemoveAttribute(position, network, String.valueOf(i));
+                }
+            }
+
+            if (network.getCellTowers() != null) {
+                position.setNetwork(network);
+            }
         }
 
         return position;
     }
 
+    private static void interpretLBSAndRemoveAttribute(Position position, Network network, String suffix) {
```

Comment:
I would probably rename this to something like `decodeCellData` or something like that.